WapTenantPublicAPI.psm1

```diff
@@ -53,7 +53,7 @@ function TestJWTClaimNotExpired {
             if ($Token.split('.').count -ne 3) {
                 throw 'Invalid token passed, run Get-WAPToken to fetch a new one'
             }
-            $TokenData = $token.Split('.')[1] | ForEach-Object -Process {
+            $TokenData = $token.Split('.')[1] | foreach-Object -Process {
```

Comment:
ForEach-Object is a cmdlet. Should be PasCal case. I guess you did a search replace?